setup.py

```diff
@@ -21,6 +21,6 @@
       include_package_data=True,
       zip_safe=False,
       setup_requires=['pytest-runner'],
-      tests_require=['pytest', 'pytest-mock', 'pytest-flask'],
+      tests_require=['pytest', 'pytest-mock', 'pytest-flask', 'pytest-aiohttp'],
       install_requires=['flasgger', 'flask', 'schematics', 'more_itertools'],
```

Comment:
just a question: is `flask` still required? And what about `aiohttp` which is frequently imported?

Reply:
We support both of them. It would probably be best to have both as optional dependencies of some kind. What do you think?

Reply:
I'd say either require both or let both optional. The latter seems better.